x-pack/plugins/security_solution/server/lib/detection_engine/rule_response_actions/types.ts

```diff
@@ -7,11 +7,18 @@
 
 import type { ParsedTechnicalFields } from '@kbn/rule-registry-plugin/common';
 
-export type Alert = ParsedTechnicalFields & {
-  _id: string;
-  agent?: AlertAgent;
-  process?: { pid: string };
-};
+export type Alert = ParsedTechnicalFields &
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  Record<string, any> & {
```

Comment:
Instead of re-defining new types here, can you `Pick<>` from the existing ones?

Reply:
I can't find a proper type that would enable me to just Pick these things. Signals are just typed as unknown, and ParsedTechnicalFields seem to be the most accurate however not fully covered. 
Do you know of any specific type we could use here for Picking?